<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
color: #EEE;
}

#Form {
background-color: qlineargradient(spread:pad, x1:0.487562, y1:0.512, x2:1, y2:0, stop:0 rgba(0, 0, 0, 242), stop:1 rgba(0, 113, 240, 238));
}

#widget {
	border-radius: 15px;
background-color: qlineargradient(spread:pad, x1:0.487562, y1:0.512, x2:1, y2:0, stop:0 rgba(0, 0, 0, 242), stop:1 rgba(0, 113, 240, 238));
color: #EEE;
}

#appHeader {
	background-color: rgba(0, 0, 0, 0.75);
	border: none;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
}

QPushButton, QComboBox {
	background: transparent;
	border-radius: 10px;
	color: white;
	border: 1px solid #999;
}
QPushButton:hover {
	border-color: red;
}
QLabel {
	background-color: transparent;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="appHeader">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 2px;
	height: 30px;
	width: 30px;
	border: 5px solid transparent;
}
QPushButton:hover {
	background-color: rgba(255, 255, 255, 0.1);
}

#appIcon {
	width: 50px;
	min-width: 50px;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <layout class="QHBoxLayout" name="header">
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="appIcon">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/src/icon.ico) 0;
width: 30px;
max-width: 40px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Auto Agent Picker</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="minimizeBtn">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color: red;
}</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Active ‚ùå</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msg">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="map">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 10px;
background-color: transparent;</string>
    </property>
    <item>
     <property name="text">
      <string>ascent</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bind</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>breeze</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fracture</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>haven</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>icebox</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pearl</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>split</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agent :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="initialiseBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Activate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setAgent">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QComboBox" name="agent">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 10px;
background-color: transparent;</string>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>astra</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>breach</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>brimstone</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>chamber</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cypher</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>deadlock</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fade</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gekko</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>harbor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>jett</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>kay/o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>killjoy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>neon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>omen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>phoenix</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>raze</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>reyna</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sage</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>skye</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sova</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>viper</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>yoru</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Map :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;</string>
    </property>
    <property name="text">
     <string>Copyright 2023 ¬©Ô∏è wissemzidi</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
